doc/cpp/recipe.rst

```diff
@@ -80,6 +80,17 @@ Recipe
 
         By default returns an empty list.
 
+    .. cpp:function:: virtual std::vector<connection> external_connections_on(cell_gid_type gid) const
+
+        Returns a list of all the **incoming** connections for `gid` from a
+        remote simulation. Each connection ``con`` should have a valid synapse
+        label ``con.dest`` on the post-synaptic target `gid`, and a valid source
+        label ``con.source.lid`` on the pre-synaptic source
+        ``con.source.gid``. See :cpp:type:`connection`.
```

Comment:
same here? also, I couldn't find the place this is pointing to

Reply:
My mistake `ext_cell_connection` is isomorphic to `connection` ;)